test/metabase/lib/drill_thru/zoom_in_bins_test.cljc

```diff
@@ -117,3 +129,100 @@
                                           [:field {:binning {:strategy :default}} (meta/id :orders :quantity)]
                                           32.5]]}]}
                 (lib/drill-thru query -1 drill)))))))
+
+;; TODO: Add a test for clicking on pivot column cells (and headers?) - but that's broken on master. See #38265.
+(deftest ^:parallel cell-click-filters-and-updates-only-one-dimension-test
+  (testing "when zooming in on one dimension, other breakouts are dropped"
```

Comment:
Where can I see that the other breakouts get dropped?

Reply:
This is badly phrased. It's more like "existing breakouts are dropped, filters are added, and new breakouts are added", so there are still breakouts in the final query.

I've updated the wording, and improved the test to show that an existing `:num-bins 10` gets turned into a `:strategy :default`.